embark-javascript-package/{{cookiecutter.repo_name}}/package.json

```diff
@@ -50,9 +50,10 @@
     "@types/async": "2.0.50",
```

Comment:
We should add `assert` here as well.

Also, we probably need 

```
    "@babel/plugin-proposal-class-properties": "7.5.5",
    "@babel/plugin-transform-runtime": "7.5.5",
    "@babel/preset-env": "7.5.5",
    "babel-jest": "24.9.0",
```
And in addition to that, a `babel.config` in the package that inherits from the root package (/cc @michaelsbradleyjr )

For the time being, the `babel.config.js` could look like this:

```
const cloneDeep = require('lodash.clonedeep');

module.exports = api => {
  const env = api.env();

  const base = {
    plugins: [
      '@babel/plugin-proposal-class-properties',
      [
        '@babel/plugin-transform-runtime',
        {
          corejs: 2
        }
      ]
    ],
    presets: ['@babel/preset-env']
  };

  if (env === 'base' || env.startsWith('base:')) {
    return base;
  }

  const node = cloneDeep(base);
  node.presets[0] = [
    node.presets[0],
    {
      targets: { node: '8.11.3' }
    }
  ];

  if (env === 'node' || env.startsWith('node:')) {
    return node;
  }

  const test = cloneDeep(node);

  if (env === 'test') {
    return test;
  }

  return {};
};
```